app/components/TokenSale/TokenSaleHeader/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+import RefreshIcon from '../../../assets/icons/refresh.svg'
+import styles from './TokenSaleHeader.scss'
+
+const TokenSaleHeader = () => (
+  <section className={styles.tokenSaleHeader}>
+    <h1>Token Sale</h1>
+    <button type="button">
+      <RefreshIcon />
+      Refresh
+    </button>
+  </section>
+)
+
+export default TokenSaleHeader
```

Comment:
Theres actually a common component (`<HeaderBar/>`) for this part of the UI (it gets rendered throughout the app) please use it and remove the files associated with TokenSaleHeader

Reply:
Perfect. Will implement.